### DISCUSSION TITLE
[Feature Request]: Add Trailer Links in Movies Database

### DISCUSSION THREAD

**Discussion by Harshit-3905** [danielschuster-muc/potter-db#190] [Ideas]
### Project

Website (Potter DB Search)

### Describe the feature you want to request

Will be adding Trailer Links Of the Harry Potter and Fantastic Beast Movies

**Comment by danielschuster-muc** [comment]
Good idea! 
Do you want to work on it?

**Comment by danielschuster-muc** [comment]
Implemented in #191.
